pkg/gardenlet/controller/managedseed/valueshelper_test.go

```diff
@@ -281,8 +279,8 @@ var _ = Describe("ValuesHelper", func() {
 						},
 					},
 					"featureGates": map[string]interface{}{
-						"ReversedVPN": false,
-						"HVPA":        true,
+						"FooFeature": true,
+						"BarFeature": true,
```

Comment:
To preserve the exact test case that is described in https://github.com/gardener/gardener/pull/7167#discussion_r1053013003

```suggestion
						"FooFeature": false,
						"BarFeature": true,
```

Reply:
Fixed.